contracts/test/TaskManager.t.sol

```diff
@@ -4,14 +4,50 @@ pragma solidity ^0.8.12;
 import "../src/FinalizerServiceManager.sol" as msm;
 import {FinalizerTaskManager} from "../src/FinalizerTaskManager.sol";
 import {BLSMockAVSDeployer} from "@eigenlayer-middleware/test/utils/BLSMockAVSDeployer.sol";
+import {BLSSignatureChecker} from "@eigenlayer-middleware/src/BLSSignatureChecker.sol";
 import {TransparentUpgradeableProxy} from "@openzeppelin/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import {BitmapUtils} from "@eigenlayer-middleware/src/libraries/BitmapUtils.sol";
+
+import { IFinalizerTaskManager } from "../src/IFinalizerTaskManager.sol";
+
+import { IBLSSignatureChecker  } from "@eigenlayer-middleware/src/interfaces/IBLSSignatureChecker.sol";
 
 contract FinalizerTaskManagerTest is BLSMockAVSDeployer {
     msm.FinalizerServiceManager sm;
     msm.FinalizerServiceManager smImplementation;
     FinalizerTaskManager tm;
     FinalizerTaskManager tmImplementation;
 
+    event NewTaskCreated(uint32 indexed taskIndex, Task task);
+
+    struct Task {
+        // L2 block number which operators are required to execute and provide proofs for
+        uint256 blockNumber;
+        // used for expiration checks
+        uint32 taskCreatedBlock;
+        // task submitter decides on the criteria for a task to be completed
+        // note that this does not mean the task was "correctly" answered
+        // task is completed when each quorumNumbers specified here
+        // are signed by at least quorumThresholdPercentage of the operators
+        // note that we set the quorumThresholdPercentage to be the same for all quorumNumbers, but this could be changed
+        bytes quorumNumbers;
+        // percentage of quorum's total stake needed to consider task completed
+        uint32 quorumThresholdPercentage;
+    }
+    
+
+    struct TaskResponse {
+        // Can be obtained by the operator from the event NewTaskCreated.
+        uint32 referenceTaskIndex;
+        // This is the response that the operator has to provide for a finalized block.
+        bytes32 blockHash;
+        // This is the response that the operator has to provide for a an executed block.
+        bytes32 storageProofHash;
+        // This is the response that the operator has to provide for a state hash at given block.
+        bytes32 pendingStateHash;
+    }
+
+
```

Comment:
why do we need this duplicated here ?

Reply:
ups, you are right, I should be able to point to those from the contract itself. good catch!

Reply:
fixed.